test/plugin/duration.test.js

```diff
@@ -139,16 +139,7 @@ describe('Clone', () => {
 })
 
 describe('Milliseconds', () => {
-  expect(dayjs.duration(500).milliseconds()).toBe(500)
```

Comment:
this PR is a little confused to me. 

Is these unit cases modification not relevant to this PR?